keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/ClusterDeployStrategy.kt

```diff
@@ -39,12 +39,16 @@ data class RedBlack(
   val maxServerGroups: Int? = 2,
   val delayBeforeDisable: Duration? = ZERO,
   val delayBeforeScaleDown: Duration? = ZERO,
+  val waitForInstancesUp: Duration? = DEFAULT_WAIT_FOR_INSTANCES_UP,
   // The order of this list is important for pauseTime based staggers
   @JsonInclude(NON_EMPTY)
   override val stagger: List<StaggeredRegion> = emptyList()
 ) : ClusterDeployStrategy() {
 
   companion object {
+    val DEFAULT_WAIT_FOR_INSTANCES_UP: Duration = Duration.ofMinutes(30)
+    val ORCA_STAGE_TIMEOUT_MARGIN: Duration = Duration.ofMinutes(5)
```

Comment:
Thoughts on just removing `ORCA_STAGE_TIMEOUT_MARGIN`?.  

That way there is no additional timeout that needs to be reasoned about beyond what is configurable (delayBeforeDisable, delayBeforeScaleDown, and  waitForInstancesUp).  I'm not sure this additional timeout margin buys us much.

Reply:
Yeah, that's a good question. I think we could get rid of it, but @emjburns suggested it so I'll defer to her for a final call.

Reply:
You know what? Was the "informed captain" on this one, I'm making the call to remove it. 😄 
The user can adjust the timeout in a more predictive manner with `waitForInstancesUp`.